boards/cc2538dk/include/periph_conf.h

```diff
@@ -76,10 +76,10 @@ static const timer_conf_t timer_config[] = {
 #define UART_0_IRQ          UART0_IRQn
 #define UART_0_ISR          isr_uart0
 /* UART 0 pin configuration */
-#define UART_0_TX_PIN       GPIO_PA1
-#define UART_0_RX_PIN       GPIO_PA0
-#define UART_0_RTS_PIN      GPIO_PD3
-#define UART_0_CTS_PIN      GPIO_PB0
+#define UART_0_TX_PIN       GPIO_PIN(0, 1)  /**< GPIO_PA1 */
+#define UART_0_RX_PIN       GPIO_PIN(0, 0)  /**< GPIO_PA0 */
+#define UART_0_RTS_PIN      GPIO_PIN(3, 3)  /**< GPIO_PD3 */
```

Comment:
In the driver implementation below it is written ` /* On the CC2538, hardware flow control is supported only on UART1 */` and that is also what the code represents. `UART_0` however, maps to `UART0` here. I'm sorry, obviously this was not intendet to fix with this PR, but could you clarify this please?

Reply:
I admit I just adapted to use `GPIO_PIN` without questioning the general config, nor consulting the data sheet on that matter. So you mean RTS and CTS pins should be removed, right?